src/battle_interface.c

```diff
@@ -873,35 +872,30 @@ void UpdateOamPriorityInAllHealthboxes(u8 priority, bool32 hideHPBoxes)
     }
 }
 
-void GetBattlerHealthboxCoords(u8 battler, s16 *x, s16 *y)
+#define TRIPLES_HEALTHBOX_DELTA     8   // need a delta bc healthboxes shifted left compared to regular players
```

Comment:
looks like this define snuck in by accident :)

Reply:
> looks like this define snuck in by accident :)

You saw nothing 🤫